src/url.js

```diff
@@ -179,8 +180,15 @@ export function serializeQueryString(params) {
     const v = params[k];
     if (v == null) {
       continue;
+    } else if (isArray(v)) {
+      for (let i = 0; i < v.length; i++) {
+        const sv = /** @type {string} */ (v[i]);
+        s.push(`${encodeURIComponent(k)}=${encodeURIComponent(sv)}`);
```

Comment:
I believe `[]` in the param name for multiple params with the same name is an unofficial server-side expectation from PHP (and maybe others). I know .NET does not have the same expectation. It just comma separates all the values regardless of the param name.